packages/integration-sdk-runtime/src/logger/registerEventHandlers.ts

```diff
@@ -71,13 +53,10 @@ export function registerEventHandlers(
 ): RegisteredEventListeners {
   const multipleResolveListener = createMultipleResolveListener(callback);
   process.on('multipleResolves', multipleResolveListener);
-  const unhandledRejectionListener = createUnhandledRejectionListener(callback);
-  process.on('unhandledRejection', unhandledRejectionListener);
```

Comment:
The behavior of `unhandledRejection` changed in Node.js 15. I talked with @ndowmon, who implemented this original event handler, and he said that we can remove it.

See: https://developer.ibm.com/blogs/nodejs-15-release-blog/

Reply:
Wdyt about removing multipleResolvesListener as well. It's deprecated.

Reply:
@ndowmon What do you think?

Reply:
go for it.